src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,101 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
 public class WorkWithFile {
+    private static final String FROM_FILE = "banana.csv";
+    private static final String RESULT_FILE = "test1.csv";
+    private static final int SIMBOLS_ARRAY = 9;
+    private static final int REST = 3;
+
+    public static void main(String[] args) {
+        WorkWithFile workWithFile = new WorkWithFile();
+        try {
+            Files.deleteIfExists(Path.of(RESULT_FILE));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't correctly clear result files after test ", e);
+        }
+        workWithFile.getStatistic(FROM_FILE, RESULT_FILE);
+    }
 
     public void getStatistic(String fromFileName, String toFileName) {
+        WorkWithFile workWithFile = new WorkWithFile();
+        String[] array = workWithFile.readFromFile(fromFileName);
+        workWithFile.writeToFile(toFileName, array);
```

Comment:
```suggestion
        String dataFromFile = readFromFile(fromFileName);
        String report = createReport(dataFromFile);
        writeToFile(toFileName, report);
```
Now implement these 3 methods